exploration/text-vs-code.md

```diff
@@ -17,49 +19,361 @@ Status: **Proposed**
+
+There are a limited number of sigils available for quoting.
+
+## Simple Messages
+
+In the Subcommittee meetings following Github discussions on Issues #493 and #499,
+the general consensus that formed for simple messages
+is that we allow them to be unquoted.
+
+("Simple messages" refers to messages consisting solely of a pattern, and thus are not complex messages.)
+
+Because the simple message pattern consists of the entire message,
+the pattern includes any leading or trailing whitespace.
+
+Given simple messages already being decided at a high level,
+the design decisions below for the proposed and alternative designs pertain specifically to complex messages.
+
+## Proposed Design
+
+Currently the syntax uses the first alternative below.
+
+## Alternatives Considered
+
+There are five candidates for handling the boundaries between code and patterns:
+
+1. Always quote non-simple patterns (current design)
+2. Never quote patterns (all whitespace is significant)
+3. Permit non-simple patterns to be quoted and trim unquoted whitespace
+4. Trim all unquoted whitespace, but do not permit quoting non-simple patterns
+5. Selectively trim patterns (all whitespace is otherwise significant)
+
+### Always Quote
+
+Pros:
+- The boundary between pattern and code is always clear.
+- The quoting reduces the number of in-pattern escapes to the open/close sequence.
+  and the placeholder sequence sigils.
+- Since the pattern is already quoted, translators never have to add pattern quotes
+  in order to add PEWs to a given pattern.
+  This also might avoid some tools forcing escaping on added quotes that are needed.
+
+Cons:
+- Requires matching open/close quotes.
+
+### Never Quote Patterns
+
+In this alternative, all non-code whitespace is significant. 
+We have to use a slightly different syntax in the example, so that
+the boundary between code and pattern works.
+>```
+>{{
+>   match {$var}
+>   {when *} This pattern has a space in front (it's between \} and This)
+>   {when other}
+>      This pattern has a newline and six spaces in front of it
+>   {when moo}This pattern has no spaces in front of it, but an invisible space at the end
+>}}
+>```
+
+Pros:
+- WYSIWYG (on steroids)
+
+Cons:
+- Probably not a serious alternative: the example
+  includes any number of obvious footguns that have to be addressed
+
+### Permit pattern quoting
+
+In this alternative, non-simple patterns are trimmed, but it is 
+possible to use quoting to separate the pattern from code (and prevent trimming)
+>```
+>{match {$var}}
+>{when   0} This has no space in front of it.
+>{when one}
+>    This has no space or newline in front of it.
+>{when few}
+>   {{ This has one space at the start and the end. }}
+>{when many}   {{ This also has one space start and end. }}
+>{when *}{{You can quote patterns even without whitespace.}}
+>```
+
+Pros:
+- Code is special instead of text.
+- Easy to use (best of both worlds?)
+
+Cons:
+- Requires one of the alternate syntaxes
```

Comment:
Given that we just ran a "beauty contest" in which these "alternate syntaxes" were preferred over the current `main` syntax by an absolute majority of the participants, this could also be listed as one of the "Pros".